client/gutenberg/extensions/presets/jetpack/utils/block-category.js

```diff
@@ -0,0 +1,12 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { getCategories, setCategories } from '@wordpress/blocks';
+
+setCategories( [
+	// Add a Jetpack block category
+	{ slug: 'jetpack', title: 'Jetpack' },
+	...getCategories().filter( ( { slug } ) => slug !== 'jetpack' ),
```

Comment:
just a style note, but it _looks_ like this is wrong at a first glance, looking like we first add the Jetpack category and then filter it out. would it be advantageous to add the existing filtered categories first _then_ add the new Jetpack category?

Reply:
We're doing this for reason - putting the `Jetpack` category at the top, before the others. I don't have any strong preferences towards this, just saying that I did it on purpose. 

cc @MichaelArestad @mapk - do you folks have any preference about the order of block categories, and the particular order in which the Jetpack category will appear in the block inserter?

Reply:
@tyxla I don't have a strong preference. I think we can go with this for now and, if core gutenberg makes a decision around this, we will follow suit.